packages/sdk/src/react/_internal/types.ts

```diff
@@ -35,3 +35,14 @@ export const AddressSchema = z.string().transform((val, ctx) => {
 export type ChainId = z.infer<typeof ChainIdSchema>;
 
 export type CollectionType = ContractType.ERC1155 | ContractType.ERC721;
+
+type TransactionStep = {
+	isExist: boolean;
```

Comment:
is exist do not make much sense as a sentence
What about
```suggestion
	pending: boolean;
```
If you want to keep the 'exist' what about just 'exist'?